orchagent/watermarkorch.h

```diff
@@ -36,8 +36,14 @@ class WatermarkOrch : public Orch
         return m_countersDb;
     }
 
+    bool isTimerEnabled()
+    {
+        return m_qWmEnabled || m_pgWmEnabled;
+    }
+
 private:
-    bool m_isTimerEnabled = false;
+    bool m_qWmEnabled = false;
+    bool m_pgWmEnabled = false;
 
```

Comment:
Instead of defining one Boolean variable for each watermark category, we may choose to use one uint64_t variable, with each category being a bit set in it.

Reply:
Now status is stored as a uint8_t